src/storage-controller/src/lib.rs

```diff
@@ -1003,9 +1021,10 @@ where
                 data_source,
                 collection_metadata: metadata,
                 extra_state,
-                // Default to 1, which is good for tables and webhooks. For
-                // ingestions, this will be updated in run_ingestion.
-                active_copies: 1,
+                // Default to empty as tables and webhooks don't run on
+                // replicas.  For ingestions, this will be updated in
+                // run_ingestion.
+                active_replicas: BTreeSet::new(),
```

Comment:
This is the only piece that confused me since the empty set represents zero, but previously the active copies was one. Is the explanation that we no longer touch this based on `DroppedId` responses?

Reply:
This now tracks how many replicas are running the collection (i.e. are interested in it's state still existing, in terms you once used with me), and tables and webhooks run on zero replicas